ph5/utilities/pformagui.py

```diff
@@ -10,23 +10,20 @@
 from ph5.core import pmonitor
 from ph5.utilities import pforma_io, watchit
 
-PROG_VERSION = '2019.14'
+PROG_VERSION = '2021.84'
 LOGGER = logging.getLogger(__name__)
 try:
-    from PySide import QtCore, QtGui
+    from PySide2 import QtCore, QtWidgets
 except Exception:
-    msg = ("\n\nNo module named PySide. "
-           "Please install PySide first, it is needed for pforma. "
-           "\n\n"
-           "If using Anaconda run 'conda install PySide'"
-           "For pip users, PySide installation instructions are available at "
-           "https://pypi.org/project/PySide/#installation.")
+    msg = ("\n\nNo module named PySide2. "
+           "Please environment_gui.yml to install conda environment"
```

Comment:
"Please use environment_gui.yml to install all required dependencies..." or something similar to that.